### ISSUE TITLE
EPIC: Blog posting ideas (https://adoptium.net/blog/)

### ISSUE THREAD

**Issue by CarmenDelgadoEclipse** [adoptium/adoptium.net-redesign#1177]
Hello community, as mentioned during the community call, we are working on getting more enterprise users onboard and aware of all the work done by the Adoptium community.

This User is familiar with mailing lists and is used to receiving community updates using a mailing list and/or RSS. That is why updating Adoptium's blog to show all the work the community is doing will be really helpful and empowering. 

**Ideas**:
- How to migrate to the OpenJDK version (talking about Adoptium).
- How to support an Open Source community 
- Do I need to be part be a member to use Adoptium technologies? What are the advantages of being a Working group member?
- Is Adoptium secure? Talk about secure-dev framework work done by the community (SBOMS, SLSA, etc.)
- Talk about the student / new contributors collaboration program and why they are important to our community.
- AQAvit verified what that means. Is this helpful for an enterprise user? How?
- Before starting a release, publish what will be released and why. - This should be part of the release process :)  
- What to talk about when preparing a CFP about Adoptium.
- Commercial support for Temurin, once we have the commercial support page published: https://github.com/adoptium/adoptium.net/pull/1808 
- Community support content. 

For more reference, this list is based on an old content stream list from 2022: "[2022 Content Streams](https://docs.google.com/spreadsheets/d/1EITCxYbyhMJRvFh5mEXi0GogVJu2PQKmHyL0QKNRT4k/edit#gid=0)" 

Please feel free to add any other topic you think will be nice to have on the blog.

The idea is to generate the same engagement as the Lightning talks and at least get **one monthly** blog post with the community's help! Thank you, everyone, in advance. 

NOTE: when creating a blog post related to this EPIC, please remember to link the issue.
